### PR TITLE
Adds edit menu for copy/paste support

### LINKED ISSUE #78: Enhancement: Allow copy paste into application
On the macOS copy of SideQuest I am unable to copy and paste a password from my password manager into the application to then send to my Quest.

I am not sure if this is macOS specific or platform wide.

### PR DESCRIPTION
**Pull request by drewbaumann**
Resolves #78

Screen Recording 2019-06-06 at 01 23 PM

Image 2019-06-06 at 1 23 40 PM

Screen Recording 2019-06-06 at 01 24 PM

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -57,16 +79,16 @@ if (!gotTheLock) {
...
-         console.log("open url")
-         event.preventDefault();
-         fs.writeFileSync(path.join(app.getPath('appData'),'SideQuest','test_output.txt'),url);
-         if(is_loaded){
-             mainWindow.webContents.send('open-url', url );
-         }else{
-             open_url = url;
-         }
-     });
+    app.on('open-url', function (event, url) {
```
Comment (drewbaumann) [inline app.js]: Corrected the indentation here.

**Comment by shaneharris** [comment]
Interesting, overall it LGTM. Is this just hidden then on windows and linux with a frameless window?

**Comment by drewbaumann** [comment]
> Interesting, overall it LGTM. Is this just hidden then on windows and linux with a frameless window?
> 
> Edit: lets find out!
> 
> All being well im doing a release tomorrow so will have this out then.

From what I understand you need to have the menu to allow for this functionality in macOS apps. 🤷🏻‍♂️. In addition to this change for macs I am considering modifying the build script as it isn't setting the icon on the macOS build.

**Comment by ANewDawn** [comment]
Will this also allow pasting in hex codes to the custom saber colours?

**Comment by drewbaumann** [comment]
> Will this also allow pasting in hex codes to the custom saber colours?

I don't see why not.

**Pull request merged**